config/action.d/iptables-ipset-proto4.conf

```diff
@@ -27,7 +27,7 @@ before = iptables.conf
 # Notes.:  command executed on demand at the first ban (or at the start of Fail2Ban if actionstart_on_demand is set to false).
 # Values:  CMD
 #
-actionstart = ipset --create f2b-<name> iphash
+actionstart = ipset --create f2b-<name> maxelem <maxelem> iphash
```

Comment:
However I'm unsure whether ipset protocol 4 (ipset v4.2) supports maxelm or the option is implemented in this way.
But since it is totally obsolete action it is OK (I'll remove it later anyway)...